nasa-gibs/worldview
# WV-2203: Prevent starting or completing a measurement when out of map bounds

## Description
Fixes WV-2203

Prior to this update, if a user clicked outside of the map bounds during a line or polygon measurement the measurement would fail with NaN. This update discards invalid clicks during a measurement which allows the measurement to continue properly.

## How To Test
Load WV, click the ruler on the bottom right, select either "Measure Distance" or "Measure Polygon".
- Test a measurement while staying within map bounds.
- Test a measurement with click(s) outside of the map bounds.
- Try to start/end a measurement with click(s) outside of the map bounds.

These steps should be repeated for both "Measure Distance" or "Measure Polygon" AND should be tested in each of the 3 projections (Arctic, Geographic, Antarctic).

@nasa-gibs/worldview
